packages/turndown-plugin-gfm/src/tables.js

```diff
@@ -205,19 +205,20 @@ function tableShouldBeSkipped(tableNode) {
   const cached = tableShouldBeSkippedCache_.get(tableNode);
   if (cached !== undefined) return cached;
 
-  const process = () => {
-    if (!tableNode) return true;
-    if (!tableNode.rows) return true;
-    if (tableNode.rows.length === 1 && tableNode.rows[0].childNodes.length <= 1) return true; // Table with only one cell
-    if (nodeContainsTable(tableNode)) return true;
-    return false;
-  }
+  const result = tableShouldBeSkipped_(tableNode);
 
-  const result = process();
   tableShouldBeSkippedCache_.set(tableNode, result);
   return result;
 }
 
+function tableShouldBeSkipped_ = (tableNode) => {
```

Comment:
Did you even try your code? That's not valid JS, is it?

Reply:
Hi,
I am sorry about this - I was in a rush, and it was late in evening.
Pushed a fix, and did some testing of the change.
Can you also please test the change - thanks

Reply:
Thanks for the update, looks good now